core/config/project_settings.cpp

```diff
@@ -101,10 +101,15 @@ const PackedStringArray ProjectSettings::_get_supported_features() {
 	features.append(VERSION_BRANCH "." _MKSTR(VERSION_PATCH));
 	features.append(VERSION_FULL_CONFIG);
 	features.append(VERSION_FULL_BUILD);
-	// For now, assume Vulkan is always supported.
-	// This should be removed if it's possible to build the editor without Vulkan.
-	features.append("Vulkan Clustered");
-	features.append("Vulkan Mobile");
+
+#ifdef VULKAN_ENABLED
+	features.append("forward_plus");
+	features.append("mobile");
```

Comment:
These names are designed to be user-friendly since they can be displayed to the user, so they should likely be "Forward Plus" etc.